startup.m

```diff
@@ -0,0 +1,48 @@
+<<<<<<< Updated upstream
```

Comment:
@salhus Is this meant to be uncommented? When I try to run the startup.m, MATLAB will error out.

Reply:
yes. sorry about that... I noticed that GitHub is adding those when I am creating a PR. I was actually trying to debug and avoid this...

Reply:
Thanks for the quick response and I have yet to come across this before so cannot provide advice on how to avoid.